tests/storage/hotplug.go

```diff
@@ -430,6 +426,32 @@ var _ = SIGDescribe("Hotplug", func() {
 		return vm
 	}
 
+	checkNoProvisionerStorageClassPVs := func(storageClass string) {
```

Comment:
I would call this storageClassName, so there is no confusion between it just being the name and the entire storage class.